### PR TITLE
HardModeSwitch is MG38 only

### PR DESCRIPTION
**Pull request by xThuby**
Fixes change from earlier commit about HardwareModeSwitch.
It's only available in MG38, so it should be in that macro block.
Also added getter from HardwareModeSwitch to coincide with the other graphics manager fields.

### CODE REVIEW

File: Nez.Portable/Utils/Screen.cs [hunk 1]
```diff
@@ -87,6 +87,12 @@ public static bool IsFullscreen
 			set => _graphicsManager.IsFullScreen = value;
 		}
 
+		public static bool HardwareModeSwitch
```
Comment (prime31) [inline Nez.Portable/Utils/Screen.cs:90]: this needs to be wrapped in compiler define

File: Nez.Portable/Assets/Tiled/Runtime/TiledMapLoader.cs [hunk 2]
```diff
@@ -75,7 +75,7 @@ public static TmxMap LoadTmxMap(this TmxMap map, XDocument xDoc)
 		private static void UpdateMaxTileSizes(TmxTileset tileset)
```
Comment (prime31) [inline Nez.Portable/Assets/Tiled/Runtime/TiledMapLoader.cs:75]: none of this is related to the hardware mode switch...

Comment (xThuby) [reply Nez.Portable/Assets/Tiled/Runtime/TiledMapLoader.cs:75]: No that's actually the previous PR I made about adding template file support to the tiledmaploader. I'm not sure why that's being included in this PR.

File: Nez.Portable/Utils/Screen.cs (hunk 1 above)
Comment (xThuby) [reply Nez.Portable/Utils/Screen.cs:91]: Done

**Pull request closed without merging**